pkg/debug/scripts/collect-info.sh

```diff
@@ -234,6 +234,11 @@ collect_pillar_backtraces()
 
     kill $pid
 
+    # collect memory backtrace
+    eve http-debug > /dev/null 2>&1
+    eve exec debug curl "http://localhost:6543/debug/pprof/heap?debug=1" | cat > "$DIR/pillar-memory-backtraces"
```

Comment:
Why not just
```
eve exec debug curl "http://localhost:6543/debug/pprof/heap?debug=1" > "$DIR/pillar-memory-backtraces"
```
?

Reply:
Yes, that's actually would work - I thought it would be better not to assume that `$DIR` is available in this mount namespace, but `>` has the same effect.